app/code/Magento/CatalogGraphQl/Model/AttributesJoiner.php

```diff
@@ -56,26 +60,70 @@ public function join(FieldNode $fieldNode, AbstractCollection $collection): void
      * Get an array of queried fields.
      *
      * @param FieldNode $fieldNode
+     * @param ResolveInfo $resolverInfo
```

Comment:
You have a mismatch here, with param. "resovleInfo" vs "resovlerInfo". I would suggest to use resolveInfo everywhere to match the classname, but either way it should be consistent